tasks/react/medium/todo-app/src/index.css

```diff
@@ -56,6 +56,20 @@ button:focus,
 button:focus-visible {
   outline: 4px auto -webkit-focus-ring-color;
 }
+.todo {
+  padding: 10px;
+  display: flex;
+  justify-content: space-between;
+}
+.tool {
+  padding: 10px;
+}
+.delete {
+  color: red;
+}
+.edit {
+  color: royalblue; 
+}
```

Comment:
Code looks good! I would also add `cursor: pointer;` to the `delete` and `edit` classes so the cursor would change to as pointer, to make them look a bit more intuitive when hovering over, but that's a minor issue. The actual functionality of the app is great.